api/samples/memval_simple.c

```diff
@@ -403,7 +421,7 @@ event_exit(void)
 DR_EXPORT void
 dr_client_main(client_id_t id, int argc, const char *argv[])
 {
-    drreg_options_t ops = {sizeof(ops), 3 /*max slots needed*/, false};
+    drreg_options_t ops = {sizeof(ops), 3 /*max slots needed*/, true};
```

Comment:
Sorry, probably should've made a specific comment on this. I saw this entry in the docs for `drreg_get_app_value`

> If `app_reg` is a dead register, `DRREG_ERROR_NO_APP_VALUE` may be returned. Set conservative in `drreg_options_t` to avoid this error

I assumed this meant conservative was required by the sample.